chapters/unitexpressions.tex

```diff
@@ -62,17 +62,21 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 \end{lstlisting}
 
 A \lstinline[language=grammar]!unit-symbol! is a non-empty string of \lstinline[language=grammar]!unit-char! characters.
-The set of allowed characters is currently not defined.
+These are restricted to \lstinline[language=grammar]!NON-DIGIT! characters.
```

Comment:
Wouldn't it be better to say this directly in the unit string grammar, thereby also incorporating the sentence above?

Reply:
Good idea, done.